### DISCUSSION TITLE
How can I restrict execution of GitHub Actions workflow on tags creation only?

### DISCUSSION THREAD

**Discussion by ainsleyrehn** [leightonavantgardehaus/.github#9] [Q&A] [answered]
Problem is when I push a new commit even with no tags it still gets triggered. I also tried using the create event but still no dice, when a new branch is created it still triggers the workflow.

Am I doing something wrong with filters? Would you have an alternative perhaps on how to trigger a workflow only when a new tag is created?

Also, if I want to filter tags that have a @ sign, is that the way to do it?

**Comment by TL8N** [comment, marked as answer]
I solved just using a full ref path: 

on:
  push:
    branches:
      - refs/tags/*

So I’m pretty sure you can then restrict to scopes like this: 

on:
  push:
    branches:
      - refs/tags/@orgname/*

I basically wanted to wildcard any tag push only, but unfortunately this is the only solution I could get it to resolute.
